ui/hooks/metamask-notifications/useProfileSyncing.ts

```diff
@@ -88,27 +86,29 @@ export function useDisableProfileSyncing(): {
   return { disableProfileSyncing, error };
 }
 
-export function useSetIsProfileSyncingEnabled(state: boolean): {
-  setIsProfileSyncingEnabled: () => Promise<void>;
+export function useSetIsProfileSyncingEnabled(): {
+  setIsProfileSyncingEnabled: (state: boolean) => Promise<void>;
```

Comment:
Fixing types here. It got past since was consumed in a JS file and did not have types.

(Prith Note) - we should try utilising `// @ts-check` at the top of JS files to get intellisense. We don't need to commit this, and can remove before committing.